### DISCUSSION TITLE
Is it possible to view S6a traffic as well using Magma?

### DISCUSSION THREAD

**Discussion by albukirky1** [magma/magma#15270] [Q&A]
I have attached an SRSran UE and eNB to magma's orchestrator and AGW version 1.7.
I'm able to see the attach procedure by tcpdump'ing eth1 (the interface I configured for s1ap).
I was wondering if there is a way to view S6a (specifically auth vectors - authentication information request & answer) and tcpdump them as well?

**Comment by panyogesh** [comment]
If your setup is using local subscriberdb (And not FeG) you can enable print_grpc_payload in subscriberdb and see the vectors.

**Reply by albukirky1** [reply]
@panyogesh Thank you for your response.
I can indeed see the AUTH vectors in the logs doing that. However, I would like to see the diameter traffic through wireshark.
I tried changing `s6a_over_grpc` to `False`, yet when I do that the attach procedure fails.
Am I on the right track? Is setting `s6a_over_grpc` to `False` (and maybe setting up a Free-Diameter service somehow) the solution?

Thanks in advance!

**Comment by panyogesh** [comment]
In case you are not using FeG deployment, Magma uses local susbcriberdb to authenticate subscriber. Only if FeG is used diameter will be used.

**Comment by panyogesh** [comment]
@albukirky1 : Please let us know if your query is addressed.

**Comment by panyogesh** [comment]
Closing the ticket. For any new issue kindly raise a new ticket.
